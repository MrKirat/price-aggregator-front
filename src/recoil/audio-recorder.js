import { atom } from 'recoil';

const isAudioRecorderEnabledState = atom({
    key: 'isAudioRecorderEnabledState',
    default: false
});

export default isAudioRecorderEnabledState;