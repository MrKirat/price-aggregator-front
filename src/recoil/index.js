export { default as isAudioRecorderEnabledState } from './audio-recorder';
export { default as isCameraEnabledState } from './camera';
export { default as searchStringState } from './search';
export { default as productListState } from './products';
export { default as isLoadingState } from './loading';