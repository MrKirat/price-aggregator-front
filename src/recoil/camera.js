import { atom } from 'recoil';

const isCameraEnabledState = atom({
    key: 'isCameraEnabledState',
    default: false
});

export default isCameraEnabledState;