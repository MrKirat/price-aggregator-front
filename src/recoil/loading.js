import { atom } from 'recoil';

const isLoadingState = atom({
    key: 'isLoadingState',
    default: false
});

export default isLoadingState;