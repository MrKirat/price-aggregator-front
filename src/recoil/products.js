import { atom } from 'recoil';

const productListState = atom({
    key: 'productListState',
    default: []
});

export default productListState;