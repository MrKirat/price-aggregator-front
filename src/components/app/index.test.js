import React from 'react';

import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import App from './index';
import { RecoilRoot } from 'recoil';

it('renders without crashing', () => {
  const output = shallow(<RecoilRoot><App /></RecoilRoot>);
  expect(shallowToJson(output)).toMatchSnapshot();
});