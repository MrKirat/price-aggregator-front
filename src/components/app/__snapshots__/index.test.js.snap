// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<ContextProvider
  value={
    Object {
      "current": Object {
        "addTransactionMetadata": [Function],
        "fireNodeSubscriptions": [Function],
        "getState": [Function],
        "replaceState": [Function],
        "subscribeToTransactions": [Function],
      },
    }
  }
>
  <Batcher
    setNotifyBatcherOfChange={[Function]}
  />
  <App />
</ContextProvider>
`;
